src/inference/src/ie_core.cpp

```diff
@@ -617,7 +617,11 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                 IE_THROW() << "Device name must not contain dot '.' symbol";
             }
             const auto& value = plugin.second;
-            PluginDescriptor desc{value.m_create_plugin_func, value.m_default_config, value.m_create_extension_func};
+            ov::AnyMap config;
+            for (const auto& it : value.m_default_config) {
+                config[it.first] = it.second;
+            }
```

Comment:
can we create `any_copy` shortcut here? To hide boilerplate code

Reply:
any_copy creates `std::map<std::string, std::string` from `ov::AnyMap`.
Here we have an opposite thing.

Reply:
I understand, that is why I'm asking about opposite function :)

Reply:
Fixed